addon/styles/addon.scss

```diff
@@ -0,0 +1,19 @@
+@import 'bower_components/bourbon/app/assets/stylesheets/bourbon';
```

Comment:
Just a note, I'd like to go with @cstolli's suggestion and start looking into https://github.com/kimroen/ember-cli-autoprefixer